PermissionsEx/permissions.yml

```diff
@@ -1,4 +1,36 @@
 groups:
+
+   optimusprime:
```

Comment:
wrong number of spaces for this node

also there's now a blank line for some reason. no good, remove it. won't compile with a blank line